webofneeds/won-owner-webapp/src/main/webapp/app/components/connection-message.js

```diff
@@ -154,14 +156,14 @@ function genComponentConf() {
             </div>
             <a ng-show="self.shouldShowRdf && self.message.get('outgoingMessage')"
                 target="_blank"
-                href="/owner/rest/linked-data/?requester={{self.encodeParam(self.ownNeed.get('uri'))}}&uri={{self.encodeParam(self.message.get('uri'))}}&deep=true">
+                href="{{self.ownerBaseUrl}}rest/linked-data/?requester={{self.encodeParam(self.ownNeed.get('uri'))}}&uri={{self.encodeParam(self.message.get('uri'))}}&deep=true">
                     <svg class="rdflink__small clickable">
                             <use xlink:href="#rdf_logo_2" href="#rdf_logo_2"></use>
                     </svg>
             </a>
             <a ng-show="self.shouldShowRdf && !self.message.get('outgoingMessage')"
                 target="_blank"
-                href="/owner/rest/linked-data/?requester={{self.encodeParam(self.ownNeed.get('uri'))}}&uri={{self.encodeParam(self.message.get('uri'))}}">
+                href="{{self.ownerBaseUrl}}rest/linked-data/?requester={{self.encodeParam(self.ownNeed.get('uri'))}}&uri={{self.encodeParam(self.message.get('uri'))}}">
```

Comment:
please change the ownerBaseUrl so it does NOT expect a trailing slash at the end, that way the links wont break even if the conf param has a trailing slash at the end

Reply:
mhm, good point
after trying a few variations, i introduced `urljoin` that handles path-concatenation correctly.